scripts/generic-ci-tests.sh

```diff
@@ -162,7 +162,7 @@ case "$TEST_SUITE" in
 
     "bok-choy")
 
-        PAVER_ARGS="-n $NUMBER_OF_BOKCHOY_THREADS --with-flaky --with-xunit"
+        PAVER_ARGS="-n $NUMBER_OF_BOKCHOY_THREADS"
```

Comment:
We removed all usages of the flaky decorator from the codebase. However, we chose not to remove the requirement so that people could use the decorator in temporary commits in order to easily verifying via multiple runs (e.g. @flaky(max_runs=20, min_passes=20)) without messing with requirements files.